packages/amazon-sumerian-hosts-core/src/core/awspack/LexFeature.js

```diff
@@ -54,7 +54,7 @@ class LexFeature extends Messenger {
       throw Error('Cannot initialize Lex feature. LexRuntime must be defined');
     }
     if (lexRuntime.config) {
-      lexRuntime.config.customUserAgent = Utils.withCustomUserAgent(
+      lexRuntime.config.customUserAgent = Utils.addCoreUserAgentComponent(
```

Comment:
Do we want to add EngineVersion to LexRuntime as well? Looks like it does in your pr description.

Reply:
Thanks! Not sure how I missed this one..Will add